recipe/meta.yaml

```
@@ -1,119 +1,121 @@
+    - pytorch ={{ compatible_pytorch }}.*=*{{ torch_proc_type }}*
+    - requests
   run:
-    # GPU requirements
-    - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}  # [pytorch_variant == "gpu"]
-    - {{ pin_compatible('numpy') }}
     - python
-    - pillow >=5.3.0,!=8.3.*
-    - pytorch
+    - pytorch ={{ compatible_pytorch }}.*=*{{ torch_proc_type }}*
+    - {{ pin_compatible('cudnn') }}           # [cuda_compiler_version != "None"]
+    - cudatoolkit {{ cuda_compiler_version }}    # [cuda_compiler_version == "11.8"]
+    - cuda-version {{ cuda_compiler_version }}   # [cuda_compiler_version != "None"]
+    - pillow >=5.3.0,!=8.3.0,!=8.3.1
     - requests
-    - jpeg  # [not win]
-    - libpng
-    - setuptools
-    # - ffmpeg
-
-# Skip test_url_is_accessible instead of hitting 20+ servers per run, since
-# each server might be occasionally unresponsive and end up failing our CI
-{% set tests_to_skip = "test_url_is_accessible" %}
-
-# 2022/08/30: Skipping the av tests (they require module 'av'): ModuleNotFoundError: No module named 'av',
-# because the av package depends on ffmpeg, which we are not requiring as a run requirement because torchvision has bugs with certain ffmpeg versions.
-{% set tests_to_skip = tests_to_skip + " or test_classes or test_feature_types or test_feature_types or test_is_valid_file or test_num_examples or test_smoke or test_str_smoke or test_transforms" %}
-# 2022/08/30: This test seems to just destroy the memory of the system.
-# they actually run on most platform except but were still hanging on linux-aarch64 and osx-arm64
-{% set tests_to_skip = tests_to_skip + " or test_forward_backward" %}
-{% set tests_to_skip = tests_to_skip + " or test_jit_forward_backward" %}
-# 2023/07/06: CIs do not have enough resources to run the full suite of model tests
-# test_extended_models
-{% set tests_to_skip = tests_to_skip + " or test_get_model or test_get_model_builder or test_get_model_weights" %}
-{% set tests_to_skip = tests_to_skip + " or test_weights_copyable or test_weights_deserializable or test_list_models" %}
-{% set tests_to_skip = tests_to_skip + " or test_get_weight or test_naming_conventions or test_schema_meta_validation" %}
-# from now on something surely needs to be disabled
-{% set tests_to_skip = tests_to_skip + " or test_transforms_jit or test_no_warn or test_pretrained_pos" %}
-{% set tests_to_skip = tests_to_skip + " or test_pretrained_kw or test_equivalent_behavior_weights" %}
-{% set tests_to_skip = tests_to_skip + " or test_multi_params or test_default_callable or test_pretrained_deprecation" %}
-# test_models
-{% set tests_to_skip = tests_to_skip + " or test_detection_model" %}
-{% set tests_to_skip = tests_to_skip + " or test_quantized_classification_model" %}  # [linux]
+    - __cuda >={{ cuda_compiler_version }}    # [cuda_compiler_version != "None"]
 
 test:
+  requires:
+    - pip
+    - pytest
+    - expecttest
+    - scipy
+    - pytest-mock
+  source_files:
+    - test
+    - gallery
```

```suggestion
    - gallery
    - pytest.ini
```
Should solve some test failures by using the ignore patterns from upstream.
https://github.com/pytorch/vision/blob/v0.18.1/pytest.ini

---

done